test/util/spec.helpers.js

```diff
@@ -0,0 +1,117 @@
+var helpers = require('../../lib/util/helpers');
+
+describe('helpers', function () {
+
+    describe('getPreviousSteps', function () {
+        var getPreviousSteps, steps;
+
+        beforeEach(function () {
+            getPreviousSteps = helpers.getPreviousSteps;
+        });
+
+        it('should expect 2 arguments', function () {
```

Comment:
Not sure it's entirely necessary that we know the arity of each function. It is useful to know that an express error handler expects 4 args, but in this case, it's probably just defensive.